include/vcpkg/base/stringview.h

```diff
@@ -86,3 +88,45 @@ namespace vcpkg
 
 VCPKG_FORMAT_AS(vcpkg::ZStringView, vcpkg::StringView);
 VCPKG_FORMAT_AS(vcpkg::StringLiteral, vcpkg::StringView);
+
+template<::size_t N>
+struct StringArray : public std::array<char, N>
```

Comment:
This is the struct that makes the concatenations possible: It inherits from `std::array` (maybe I should make the array a class member rather than inheriting from it [react with 👍if you want me to do that]).

I'm unsure where (in terms of namespace and header file) to put this because there might be other cases where this struct is useful so I didn't want to put it into `messages.h`.

Reply:
I prefer to see the std::array be a member rather than use inheritance, since several of the std::array members, including size, do the wrong thing.